src/backends/backend/python/python_interpreter.cc

```diff
@@ -744,9 +749,22 @@ TRITONBACKEND_ModelInstanceExecute(
   ni::ExecuteResponse execute_response;
 
   // Perform inference on the Python side
-  const auto err = instance_state->stub->Execute(
+  const auto status = instance_state->stub->Execute(
```

Comment:
What if there were errors in some of the requests and so you have already sent error responses for those? How does the python code know about that? The lifecycle of requests and responses is important and is documented in the Backend section of the docs and in tritonbackend.h.
We should discuss the overall handling of this lifecycle as what is here now seems to be incorrect.

Reply:
I release all the requests if an error happens during inference.